homeassistant/util/logging.py

```diff
@@ -43,15 +43,12 @@ def __init__(self, loop, handler):
         self.handleError = handler.handleError
         self.format = handler.format
 
+        self._thread.start()
+
     def close(self):
         """Wrap close to handler."""
         self.emit(None)
 
-    def open(self):
```

Comment:
No, Handler has no open. I miss anderstand the FileHandler construct with _open...